pkg/velerorunner/backup.go

```diff
@@ -178,3 +183,64 @@ func (t *Task) getReplicatedBackup() (*velero.Backup, error) {
 
 	return nil, nil
 }
+
+// Delete the running restic pod in velero namespace
+// This function is used to get around mount propagation requirements
+func (t *Task) bounceResticPod() error {
+	// If restart already completed, return
+	if t.Phase != WaitOnResticRestart && t.Phase != Started {
+		return nil
+	} else if t.Phase == WaitOnResticRestart {
+		t.Log.Info("Waiting for restic pod to be ready")
+	}
+
+	// Get client of source cluster
+	client, err := t.getSourceClient()
+	if err != nil {
+		return err
+	}
+
+	// List all pods in velero namespace
+	list := corev1.PodList{}
+	err = client.List(
+		context.TODO(),
+		&k8sclient.ListOptions{
+			Namespace: VeleroNamespace,
```

Comment:
I'm not sure if it would make any performance difference, but if we're really trying to clamp down on apiserver traffic at some point in the future we could perhaps query with a labelselector for `name=restic`. Don't think there's a good reason to switch this around at this point.

Reply:
^ This kind of thing would be _very_ low on the priority list though... we are generating expontentially more apiserver traffic through validations compared to this one time query. Was really just curious if there was a label here.

Reply:
Ah yeah I actually don't know if restic has a label, I can check.